kong/plugins/zipkin/schema.lua

```diff
@@ -1,14 +1,16 @@
-local Errors = require "kong.dao.errors"
+local typedefs = require "kong.db.schema.typedefs"
 
 return {
+	name = "zipkin",
 	fields = {
-		http_endpoint = { required = true, type = "url" },
-		sample_ratio = { default = 0.001, type = "number" },
+		{ config = {
```

Comment:
I don't recall seeing a `config` sub-table before. Is this correct?

Reply:
Yes, that's the new "plugin schema format". See for example the `key-auth` schema: https://github.com/Kong/kong/blob/next/kong/plugins/key-auth/schema.lua